run.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+PATH=target/debug:"$PATH" cargo run --bin foxbox "$@"
```

Comment:
I designed this to be used only as a development aid. As you said before, the PATH handling should be part of the deployment setup, and so I thought that we don't need to handle `--release`.

I'm totally fine working on this a little more (but I'm in PTO tonight :p) but I want to keep this run.sh very simple initially. I don't want a full-featured runner for now.